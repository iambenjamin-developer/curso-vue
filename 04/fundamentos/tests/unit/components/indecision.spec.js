import { shallowMount } from "@vue/test-utils";
import Indecision from "@/components/Indecision.vue";

describe('Indecision Component', () => {

    let wrapper;

    beforeEach(() => {

        wrapper = shallowMount(Indecision);
    });



    test('Debe hacer match con el snapshot', () => {

        expect(wrapper.html()).toMatchSnapshot();
    })

});