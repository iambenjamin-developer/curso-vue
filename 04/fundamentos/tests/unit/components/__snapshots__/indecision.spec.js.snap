// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Indecision Component Debe hacer match con el snapshot 1`] = `
<h1>Indecision</h1>
<!--v-if-->
<div class="bg-dark"></div>
<div class="indecision-container"><input type="text" placeholder="Haceme una pregunta">
  <p>Recorda terminar con un signo de interrogaci√≥n(?)</p>
  <!--v-if-->
</div>
`;
